modules/ROOT/pages/cassandra/cassandra-studio.adoc

```diff
@@ -0,0 +1,111 @@
+= Cassandra-Connector.adoc - Studio - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
```

Comment:
Should add ":imagesdir: ../../assets/images/" immediately after endif::[] so you can see images in Git

Reply:
Oops, got carried away and took this out. It's back now.